pkg/terraform/exec/plugins/vsphereprivate/config.go

```diff
@@ -0,0 +1,571 @@
+// Modified From:
+// https://github.com/terraform-providers/terraform-provider-vsphere/blob/master/vsphere/config.go
+
+/* NOTES:
+ * - Removed required functions or sections of functions that
+ *   require terraform-provider-vsphere internal helpers.
+ * - Changed package name
+ * - For the ova import we will not use the pbm client or the rest client
+ * - go linting issues
+ */
```

Comment:
hmm.. let's keep `//`

Reply:
> sections of functions that require terraform-provider-vsphere internal helpers.

we should be able to use terraform-providers-vsphere/vsphere/config.go which can use the internal functions?